blockstack/lib/fast_sync.py

```diff
@@ -612,44 +620,44 @@ def logerr(s):
         log.exception(e)
         return False
 
-    num_signatures = 0
-    ptr = file_size
-    signatures = []
-
-    with open(import_path, 'r') as f:
-        info = fast_sync_inspect( f )
-        if 'error' in info:
-            logerr("Failed to inspect snapshot {}: {}".format(import_path, info['error']))
-            return False
-
-        signatures = info['signatures']
-        ptr = info['payload_size']
+    if num_required > 0:
```

Comment:
This check shouldn't be necessary -- the code will work (and emit helpful log output) even if `num_required` is zero.